lib/hours.php

```diff
@@ -665,646 +626,598 @@ function import_hours() {
+
+							// This was print_r($term[term]).
+							$termId = $term[ term ][ id ];
+
+							echo 'Adding Holiday - ' . $day . ' / ' . $formatDate . ' = ' . $val . '<br>';
+
+							if ( $termId == 0 || $termId == '' ) {
 								echo "<b>MISSING Term for: $termDate</b><br>";
 							} else {
-								// OK to insert
-							
-							
-								// Insert Post
+								// OK to insert.
+								// Insert post.
 								$args = array(
 									'post_title' => $day,
 									'post_parent' => $termId,
 									'post_type' => 'hours',
-									'post_status' => 'publish'
+									'post_status' => 'publish',
 								);
-								
-								$postId = wp_insert_post($args);
-								
-								$val = str_replace(":00", "", $val);
-								
-								add_post_meta($postId, "start", $formatDate);						
-								add_post_meta($postId, "description", $val);
-								add_post_meta($postId, "tag", $tag);
-								
+
+								$postId = wp_insert_post( $args );
+
+								$val = str_replace( ':00', '', $val );
+
+								add_post_meta( $postId, 'start', $formatDate );
+								add_post_meta( $postId, 'description', $val );
+								add_post_meta( $postId, 'tag', $tag );
+
 							}
-							
-						}
-						
-						
-						
-					}
+}
+}
 				}
 			}
-			
-			
-			
-		} else {
-			// catch hidden extra areas
-			if ($started && $item[A] == "") {
+} else {
+			// Catch hidden extra areas.
+			if ( $started && $item[ A ] == '' ) {
 				return;
 			}
 		}
 	}
-	
-}	
 
-function semesterNameToId($name, $locationId) {
+}
+
+function semesterNameToId( $name, $locationId ) {
 	$args = array(
 		'post_type' => 'hours',
 		'posts_per_page' => -1,
 		'search_prod_title' => $name,
 		'post_parent' => $locationId,
 	);
-	
+
 	add_filter( 'posts_where', 'title_filter', 10, 2 );
 	$semList = new WP_Query( $args );
 	remove_filter( 'posts_where', 'title_filter', 10, 2 );
-	
-	if ($semList->have_posts()) {
+
+	if ( $semList->have_posts() ) {
 		$semList -> the_post();
-		
+
 		return get_the_ID();
-		
-	};	
-	
+
+	};
+
 	return 0;
 }
 
-function nameLookup($name) {
-	switch(strtolower($name)) {
-		case "barker":	
-			$baseName = "Barker Library";
+function nameLookup( $name ) {
+	switch ( strtolower( $name ) ) {
+		case 'barker':
+			$baseName = 'Barker Library';
 			break;
-		case "dewey":	
-			$baseName = "Dewey Library";
+		case 'dewey':
+			$baseName = 'Dewey Library';
 			break;
-		case "hayden":	
-			$baseName = "Hayden Library";
+		case 'hayden':
+			$baseName = 'Hayden Library';
 			break;
-		case "rotch":	
-			$baseName = "Rotch Library";
+		case 'rotch':
+			$baseName = 'Rotch Library';
 			break;
-		case "lewis":	
-			$baseName = "Lewis Music Library";
+		case 'lewis':
+			$baseName = 'Lewis Music Library';
 			break;
-		case "maihaugen":
-		case "miahaugen":
-		case "maihuagen":
-		case "miahuagen":
-			$baseName = "Maihaugen Gallery";
+		case 'maihaugen':
+		case 'miahaugen':
+		case 'maihuagen':
+		case 'miahuagen':
```

Comment:
I'm not entirely sure why this was built this way - and I'm not sure how easily we could use something like word stemming or similar technique to catch misspelled locations.

It should be pointed out that I'm fairly certain this file is unused, so the whole thing may be able to be removed in the near future. Given that, I'm not particularly worried about refactoring this beyond what's necessary to to get validators to pass.